ile/src/handler.cpp

```diff
@@ -144,14 +144,14 @@ int main(int _argc, char **argv)
     memset(msg_info_buf, 0x00, msg_info_buf_size);
     if (' ' == qualified_msg_file[0])
     {
-      DEBUG("Message not from message file\n");
+      DEBUG_WARNING("Message not from message file\n");
```

Comment:
I believe this should also just be an info message.

```suggestion
      DEBUG_INFO("Message not from message file\n");
```